api/v1beta1/barbican_webhook.go

```diff
@@ -141,13 +141,6 @@ func (r *BarbicanSpec) ValidateCreate(basePath *field.Path) field.ErrorList {
 			allErrs = append(allErrs, field.Required(basePath.Child("PKCS11"),
```

Comment:
we should have the same check on `ValidateUpdate` if someone enables P11 as a day2.

Reply:
Done